### PR TITLE
Add footer links, routes and placeholder html

### PR DESCRIPTION
**Pull request by arthurashman**
## Description of change
Adds links to the footer for:
Cookies
Contact
Feedback
Privacy policy
Accessibility statement 
Terms and conditions 

This follows design of Crime Apply Prototype 
Links to the same external pages for cookies and terms and conditions as Civil Apply Prototype (cookies is a generic gov.uk cookies page - their staging env goes to a service specific one)

Adds routes and html files for contact, feedback, accessibility - content needs creating and updating. 

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-8

### Before changes:
image

### After changes:
image

### CODE REVIEW

File: app/views/about/accessibility.en.html.erb [hunk 1]
```diff
@@ -0,0 +1,157 @@
+<%# CONTNENT NEEDS CREATING %>
+
+<% title 'Privacy policy' %>
```
Comment (zheileman) [inline app/views/about/accessibility.en.html.erb:3]: Are all pages privacy policy content, on purpose? 🤣 

If it is a placeholder, wouldn't be better to have at least their title correspond with what the page is suppose to be, and then as body something like "Placeholder. Copy to be decided." or similar?

Having all them the same is confusing.

File: app/views/layouts/_footer_links.html.erb [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+  </li>
+
+  <li class="govuk-footer__inline-list-item">
+    <%# Content needs creating %>
+
+    <%= link_to 'Feedback', 'about/feedback', class: 'govuk-footer__link' %>
+  </li>
+
+  <li class="govuk-footer__inline-list-item">
+    <%= link_to 'Privacy policy', 'about/privacy', class: 'govuk-footer__link' %>
```
Comment (zheileman) [inline app/views/layouts/_footer_links.html.erb:23]: We can use the route helpers for the links, i.e. `about_privacy_path`

**Review by zheileman — APPROVED** [review]
👍

File: app/views/layouts/_footer_links.html.erb (hunk 2 above)
Comment (arthurashman) [reply app/views/layouts/_footer_links.html.erb]: 👍 Thank you

File: app/views/about/accessibility.en.html.erb (hunk 1 above)
Comment (arthurashman) [reply app/views/about/accessibility.en.html.erb]: 👍 Good idea. Done!

**Review by kennyevil — APPROVED** [review]

**Pull request merged**